crates/rolldown_utils/src/light_guess.rs

```diff
@@ -0,0 +1,129 @@
+// Port from https://github.com/7086cmd/mime_more/blob/main/src/light_guess.rs
+use mime::Mime;
+use phf::{phf_map, Map};
+use std::str::FromStr;
+
+pub static MIME_TYPES: Map<&'static str, &'static str> = phf_map! {
```

Comment:
`mime_guess` used `phf` too. Guess we don't need to maintain this? https://github.com/abonander/mime_guess/blob/master/src/impl_phf.rs

Reply:
Yes, I think. phf provides a better performance (O(1)), but about 60 items is ambiguous to tell the performance of phf and match apart. But when we need other MIME types, using phf can be better.

Reply:
phf is not authored by me :) I just learned mime_guess.

Reply:
I mean `mime_guess` is already used `phf` too. we don't need to maintain our own version of `MIME_TYPES `.

https://github.com/abonander/mime_guess/blob/master/src/impl_phf.rs